src/scaffoldmaker/meshtypes/meshtype_3d_cecum1.py

```diff
@@ -0,0 +1,1059 @@
+        '''
+        :param parameterSetName:  Name of valid parameter set for option Scaffold, or None for default.
+        :return: ScaffoldPackage.
+        '''
+        if parameterSetName:
+            assert parameterSetName in cls.getOptionScaffoldTypeParameterSetNames(optionName, scaffoldType), \
+                'Invalid parameter set ' + str(parameterSetName) + ' for scaffold ' + str(scaffoldType.getName()) + ' in option ' + str(optionName) + ' of scaffold ' + cls.getName()
+        if optionName == 'Central path':
+            if not parameterSetName:
+                parameterSetName = list(cls.centralPathDefaultScaffoldPackages.keys())[0]
+            return copy.deepcopy(cls.centralPathDefaultScaffoldPackages[parameterSetName])
+        if optionName == 'Ileocecal junction':
+            if not parameterSetName:
+                parameterSetName = list(cls.ostiumDefaultScaffoldPackages.keys())[0]
+            return copy.deepcopy(cls.ostiumDefaultScaffoldPackages[parameterSetName])
+        assert False, cls.__name__ + '.getOptionScaffoldPackage:  Option ' + optionName + ' is not a scaffold'
+
+    @classmethod
+    def checkOptions(cls, options):
+        if not options['Central path'].getScaffoldType() in cls.getOptionValidScaffoldTypes('Central path'):
+            options['Central path'] = cls.getOptionScaffoldPackage('Central path', MeshType_1d_path1)
+        if not options['Ileocecal junction'].getScaffoldType() in cls.getOptionValidScaffoldTypes('Ileocecal junction'):
+            options['Ileocecal junction'] = cls.getOptionScaffoldPackage('Ileocecal junction', MeshType_3d_ostium1)
+        for key in [
+            'Number of segments',
+            'Refine number of elements around',
+            'Refine number of elements along',
+            'Refine number of elements through wall']:
+            if options[key] < 1:
+                options[key] = 1
+        for key in [
+            'Number of elements around tenia coli',
+            'Number of elements around haustrum',
+            'Number of elements along segment',
+            'Number of elements through wall',
+            'Start inner radius',
+            'Start inner radius derivative',
+            'End inner radius',
+            'End inner radius derivative',
+            'Corner inner radius factor',
+            'Haustrum inner radius factor',
+            'Segment length end derivative factor',
+            'Segment length mid derivative factor',
+            'Number of tenia coli',
+            'Start tenia coli width',
+            'Start tenia coli width derivative',
+            'End tenia coli width',
+            'End tenia coli width derivative',
+            'Ileocecal junction angular position degrees',
+            'Ileocecal junction position along factor',
+            'Tenia coli thickness',
+            'Wall thickness']:
+            if options[key] < 0.0:
+                options[key] = 0.0
+
```

Comment:
Implement a function like that in heartatria to transfer settings (ostium wall thickness etc.) to the ostium:
```
    @classmethod
    def updateSubScaffoldOptions(cls, options):
```
Call this function in each of getDefaultOptions, checkOptions, generateBaseMesh as in heartatria1.
This ensures the dependent ostium options are already up-to-date.
I'm not sure if the number around can be set, but see if the logic can be moved here.

Reply:
I made the changes for ostium wall thickness, but the number of elements around ostium is only determined much later after we have created the cecum segments and identified the tracksurface where ostium sits, so I don't think it makes sense to move it to this function.  Also, the number of elements around is derived and updated right before I make the ostium so it will ensure that the option is up-to-date:
ostiumSettings['Number of elements around ostium'] = len(innerEndPoints_Id)